samples/stock/demo/step-line/demo.js

```diff
@@ -2,26 +2,78 @@
 
     // Load the dataset
     const data = await fetch(
-        'https://demo-live-data.highcharts.com/aapl-c.json'
+        'https://cdn.jsdelivr.net/gh/highcharts/highcharts@a9dcb12aad/samples/data/investment-simulator.json'
     ).then(response => response.json());
 
     // Create the chart
     Highcharts.stockChart('container', {
 
+        chart: {
+            spacingRight: 25
+        },
+
         rangeSelector: {
             selected: 1
         },
 
         title: {
-            text: 'AAPL Stock Price'
+            text: 'Recurring stock investment simulator'
```

Comment:
Q: How about simply an `Investment simulator`? It doesn't have to be stock, might be cash, or other asset, right?